core/src/test/java/org/frankframework/pipes/IsUserInRolePipeTest.java

```diff
@@ -0,0 +1,304 @@
+package org.frankframework.pipes;
+
+import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.core.ISecurityHandler;
+import org.frankframework.core.PipeForward;
+import org.frankframework.core.PipeLineSession;
+import org.frankframework.core.PipeRunResult;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * IsUserInRolePipe unit tests.
+ *
+ * @author <Laurens Mäkel>
+ */
+public class IsUserInRolePipeTest extends PipeTestBase<IsUserInRolePipe> {
```

Comment:
Please remove the `public` modifier for all test methods and class, when using JUnit 5.